nengo_dl/tests/test_simulator.py

```diff
@@ -1437,7 +1437,7 @@ def test_synapse_warning(Simulator):
 
     def does_warn(n_steps=1, as_dict=True):
         with Simulator(net, unroll_simulation=1) as sim:
-            with pytest.warns(UserWarning) as rec:
+            with pytest.warns(None) as rec:
```

Comment:
None means it will capture warnings in `rec`, but not raise an error if there are no warnings, which is what we want here I think (can't see the diff well on phone).

Reply:
Huh, interesting. I think you could also use the underlying `WarningsCatcher` instead, but this keeps the diff shorter so sounds good to me.